kymatio/backend/base_backend.py

```diff
@@ -1,11 +1,14 @@
 
 
 class FFT:
-    def __init__(self, fft, ifft, irfft, type_checks):
+    def __init__(self, fft, rfft, ifft, irfft, type_checks, real_check, complex_check):
         self.fft = fft
```

Comment:
Yes its used.

Reply:
You have a point.

Reply:
It almost seems like we don't even need the inverse variable in the forward call now. 

R2C is rfft
C2C is ifft
C2R is irfft

It almost seems like (in the forward call) we could just get rid of the inverse if we get rid of the fft, but we should prolly keep the inverse variable. Should I have it throw an error in that case? Thoughts @eickenberg ?

Reply:
Yeah I wouldn't do that. Better to be explicit here. Someone looking at `fft(x, 'C2C')` is not going to think that it's obviously an *inverse* FFT. I'm also not a 100% into specifying all these things as arguments (especially since we have a class structure now). Maybe that should go away at some point.

Reply:
wait what? `fft(x, 'C2C')` is of "ifft" nature?

Reply:
no, it's forward by default